Integrations/integration-ArcherRSA.yml

```diff
@@ -2610,7 +2610,7 @@ script:
         } else {
             fieldsToValue = JSON.parse(dataArgs.fieldsToValues);
         }
-        var incidentId = dataArgs.incidentId;
+        var incidentId = dataArgs.incidentId.toString();
```

Comment:
this may be undefined, convert the value to string inside the condition in the following line